### ISSUE TITLE
Use old-style argument names from GATK

### ISSUE THREAD

**Issue by magicDGS** [magicDGS/ReadTools#346]
Related with #340 

Before getting into ReadTools version 2.0.0, updating to the latest master of GATK is going to change a lot of argument names that are in current use. To ensure that nothing is broken and the CLI is consistent with the current version for our users, we should:

- [x] 1. Modify all the tests to use hard-coded argument names - this will ensure that minimum modifications in GATK or in our own CLI does not break compatibility.
- [x] 2. Modify all usages in our own code of `StandardArgumentDefinitions`
- [x] 3. Add an hacked `StandardArgumentDefinitions` in the same package as GATK to get the correct in-use arguments for version 1 (see #364) and be sure that arguments in ReadTools that are in the GATK repository are all contain in that class.
- [x] 4. Add a hacked `ReadFilterArgumentDefinitions` before updating to the latest GATK4 (depends on acceptance of https://github.com/broadinstitute/gatk/pull/4103)
- [x] 5. Find all usages of argument collections from GATK if any, and substitute by our own argument collections if required.
- [x] 6. Find all usages of plugins from GATK if any, and substitute by our own classes/arg collection if it contains any difference.
- [x] 7. Find all usages of program groups from GATK if any, and substitute by our own if it contains any difference.
- [ ] For version 2.0.0, remove the hack to use the GATK released kebab-case arguments (we will require to fix #340 to be consistent in our own CLI)

This is high priority, because I would like to include the released GATK4 in January 9th ASAP, to depend on an already release jar and not an SNAPSHOT.

**Comment by magicDGS** [comment]
We should also contribute to GATK before their release to include a way to maintain the `CommandLineProgram` arguments to the current ones.

**Comment by magicDGS** [comment]
Edited for new tasks on this regard.

**Comment by magicDGS** [comment]
Two first tasks were done in #354

**Comment by magicDGS** [comment]
No usages after #354 of argument collections (5th task):
* `org.broadinstitute.hellbender.cmdline.argumentcollections`
* `org.broadinstitute.hellbender.cmdline.ExomeStandardArgumentDefinitions`

**Comment by magicDGS** [comment]
6th task: the `GATKReadFilterPluginDescriptor` is used, but the arguments are configurable by an argument collection. Only `TrimReadsFilterPluginArgumentCollection` was using argument definitions from GATK, but not any longer after #354 - it requires an update in the doc, but that's all (will fix in #365)

**Comment by magicDGS** [comment]
Closing in favor of #340 - everything except the kebab-case is included (and that's part of the version 2.0.0 epic)
